organization/_posts/2022/2022-03-24-generators.md

```diff
@@ -1,9 +1,14 @@
-Minutes of the HSF Generator WG Meeting No:18 Thursday 24 March 2022 https://indico.cern.ch/event/1141896/?note=192096
+---
+title: "HSF Generator WG Meeting #18, 24 March 2022"
+layout: plain_toc
+---
+
+Agenda: https://indico.cern.ch/event/1141896/
```

Comment:
Can you put the URL in angled brackets: `<https://indico.cern.ch/event/1141896/>` - otherwise Jeykll doesn't render it as a link?